utils/releaser/src/ReleaseWorker/ReleaseCandidate/TestYourBranchLocallyReleaseWorker.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\ReleaseCandidate;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+use Symplify\MonorepoBuilder\Release\Message;
+
+final class TestYourBranchLocallyReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Test your branch locally - running composer-dev, standards and tests - this might take a few minutes"';
```

Comment:
what is the purpose of single double quote character ?
```suggestion
         return 'Test your branch locally - running composer-dev, standards and tests - this might take a few minutes';
```